### ISSUE TITLE
Problem with the graphic point navigation

### ISSUE THREAD

**Issue by miguelmoreto** [miguelmoreto/labcontrole#17]
If the user click on the simulation graphic, this feature adds a circle over the graphic that can be used to view the respective coordinate and navigate throughout all the points using the mouse scroll.

But this feature works only if the user does not use de zoom function. 

Using the binary version of Labcontrole, after the user used the zoom for the first time this new feature no longer works afterwards. Only after re-open the program.

@andidevel I appreciate if you can check this. But only if you have time, it is not urgent.

**Comment by andidevel** [comment]
In fact, this function works only while not "zooming" or "panning", but, if you "uncheck" the button you can use the feature again, at least, in my tests, running on Linux. I'll check this, though.

**Comment by andidevel** [comment]
Done! I did a new pull request, #18, fixing the problem.

Actually, it wasn't a **"problem"**. I did that to avoid mouse event conflict with NavigationToolBar super class, since I didn't know how this class works very well. But I agree, it was kind annoying use zoom or pan and lose the navigation util release the buttons (exiting from Zoom/Pan modes) on navigation toolbar.

**Comment by miguelmoreto** [comment]
Thanks a lot @andidevel !

I'm working with porting the LabControle2 to Python 3 and pyQt5. It is almost done.

Best regards,

Moreto
